lib/cached-request.js

```diff
@@ -138,6 +139,7 @@ CachedRequest.prototype.cachedRequest = function () {
   key = this.hashKey(key);
 
   //Open headers file
+  mkdirp(this.cacheDirectory);
```

Comment:
I missed the fallback to `MakeRequest` on `ENOENT` - Code has been fixed to reflect that. 
Thanks.